foundations/javascript_basics/object_basics.md

```diff
@@ -119,7 +129,8 @@ However, there are two key differences with this array method:
 
 ```js
 const arr = [1, 2, 3, 4, 5];
-arr.reduce((total, currentItem) => total * currentItem, 1); // Outputs 120;
+console.log(arr.reduce((total, currentItem) => total * currentItem, 1)); // Outputs 120;
+console.log(arr) // Outputs [1, 2, 3, 4, 5], original array is not affected 
```

Comment:
```suggestion
const productOfAllNums = arr.reduce((total, currentItem) => {
  return total * currentItem;
}, 1);
console.log(productOfAllNums); // Outputs 120;
console.log(arr) // Outputs [1, 2, 3, 4, 5]
```
I think consistency between examples is more important here.
It's not vital for us to demonstrate how the return value of `.reduce` can be passed directly to `console.log` without the need for an intermediary variable.
In fact, the addition of an intermediary variable here IMHO significant improves how understandable the code is in terms of what it's doing.
Doing it like this as well (with the `{ }` in the callback) also means the code block doesn't need to scroll horizontally.